### PR TITLE
Add VGG16

### PR DESCRIPTION
**Pull request by Ahleroy**
Add  the VGG16 file.
Not fully functionnal, some help is welcomed.

### CODE REVIEW

**Comment by yhenon** [comment]
@Ahleroy Thanks for this PR - I've merged your changes in, although I didn't diretly pull in the PR. Perhaps I can add a `contributors` file to credit you properly ?

**Comment by Ahleroy** [comment]
@yhenon Thank you for the update. I wanted to push an updated version of the VGG16 but if your modification works then it is find by me.

This would be very nice from you to add this contributors file :)

**Comment by Ahleroy** [comment]
The update of the VGG16 model does not work.

The output of the `get_output_length` function in the `vgg.py` file should be :

` return input_length//16` and not  `return input_length/16`

The last convolution block should be time distributed, right ?

**Comment by yhenon** [comment]
Hey, thanks for looking over the code. You are correct on point 1 (this is an oversight on my part as I didnt test with python3). Regarding point 2, I think my implementation is correct as it matches the implementation in the official repo here  https://raw.githubusercontent.com/rbgirshick/py-faster-rcnn/master/models/pascal_voc/VGG16/faster_rcnn_end2end/test.prototxt

What do you think?

**Comment by zeehuilin** [comment]
Hi,when I train the Keras_frcnn with VGG netwok, I got this error:
Caused by op u'time_distributed_3/keras_learning_phase', defined at:
  File "train_frcnn.py", line 130, in <module>
    classifier = nn.classifier(shared_layers, roi_input, C.num_rois, nb_classes=len(classes_count), trainable=True)
  File "/home/linzehui/keras/keras-frcnn/keras_frcnn/vgg.py", line 113, in classifier
    out = TimeDistributed(Dropout(0.5))(out)

And , Keras_rcnn with resnet50 work well. But the training speed is so slow. I only trained 50 iterations a day with GPU(on the same computer, I trained tf_frcnn in a day).In addition, The dataset I use is the VOC dataset. Can anyone help me?

**Comment by zeehuilin** [comment]
Hi,the error was sloved. I got rid of the "out = TimeDistributed(Dropout(0.5))(out)" in vgg.py. Now the program can train normally, but I don't know what the consequences are.
In additation, the speed is still slow.
